Sources/Views/Products/Detail/ProductAttributes/AttributeView.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  AttributeView.swift
+//  OpenFoodFacts
+//
+//  Created by Alexander Scott Beaty on on 17/10/2020.
+//  Copyright © 2020 Alexander Scott Beaty. All rights reserved.
+//
+
+import UIKit
+import BLTNBoard
+import Kingfisher
+import Cartography
+
+@IBDesignable class AttributeView: UIView {
+
+    @IBOutlet weak var iconImageView: UIImageView!
+    @IBOutlet weak var descriptionShort: UITextView!
+
+    var attribute: Attribute?
+
+    func configure(_ attribute: Attribute) {
+        self.layer.cornerRadius = 5
+        self.attribute = attribute
+        setIconImageView(imageURL: attribute.iconUrl)
+        if let label = attribute.name, let description = attribute.descriptionShort ?? attribute.title {
+            let text = AttributedStringFormatter.formatAttributedText(label: label, description: description)
+            descriptionShort.attributedText = text
+        }
+    }
+
+    static func loadFromNib() -> AttributeView {
+        let nib = UINib(nibName: "AttributeView", bundle: Bundle.main)
+        // swiftlint:disable:next force_cast
+        let view = nib.instantiate(withOwner: self, options: nil).first as! AttributeView
+        return view
+    }
+
+    func setIconImageView(imageURL: String?) {
+        guard let icon = imageURL,
+              let url = URL(string: icon)
+        else {
+            iconImageView.isHidden = false
+            return
+        }
+        iconImageView.kf.indicatorType = .activity
+        iconImageView.kf.setImage(with: url)
+        iconImageView.isHidden = false
+    }
+
+    var bulletinManager: BLTNItemManager!
+
+    deinit {
+        if bulletinManager != nil {
+            if bulletinManager.isShowingBulletin {
+                bulletinManager.dismissBulletin(animated: true)
+            }
+            bulletinManager = nil
+        }
+    }
+}
+
+protocol formatAttributedString {
```

Comment:
I made this protocol to expose the private methods used  to format attributed strings found in IngredientsAnalysisView.swift

They are almost exactly the same and now they can be used by anyone else.